examples/benchmarks/main.cpp

```diff
@@ -10,31 +10,26 @@
 
 #include "tiny_dnn/tiny_dnn.h"
 
-using namespace tiny_dnn;
-using namespace std;
-
-int main(int argc, char **argv) {
-  CNN_UNREFERENCED_PARAMETER(argc);
-  CNN_UNREFERENCED_PARAMETER(argv);
-  models::alexnet nn;
+int main() {
+  tiny_dnn::models::alexnet nn;
```

Comment:
Aren't we following `using namepsace tiny_dnn;` in examples or sample code snippets ? I have always seen it everywhere whenever there is a sample boilerplate code in docs....